### ISSUE TITLE
System information not reporting SPI ram info ESP32-S3

### ISSUE THREAD

**Issue by ThinkTransit** [jczic/ESP32-MPY-Jama#38]
Hi,

Great software!

I noticed when using an ESP32-S3 32MB flash and 8MB RAM, the system information says SPIRAM available no, should that be yes?

<img width="1473" alt="image" src="https://user-images.githubusercontent.com/74940387/231330027-5afc96b1-dee3-4765-807e-3d9c174a26fd.png">

**Comment by jczic** [comment]
Hi @ThinkTransit,
Ok I see, the term of spiram does not appear in the implementation.
I checked for your GENERIC_S3 type card and indeed, the configuration file does not directly include the use of SPIRAM so that must be why. I'm only doing this check for now...
( https://github.com/micropython/micropython/blob/master/ports/esp32/boards/GENERIC_S3/sdkconfig.board )

**Comment by ThinkTransit** [comment]
Ok that makes sense. Maybe it might be useful to add total memory to the system information, gc.mem_alloc() + gc.mem_free().

Just a suggestion.

**Comment by jczic** [comment]
Exact! However there is at the bottom left the gauge with these 2 informations but all the fonts seem to be unstaggered on your system :(

**Comment by anglerfish27** [comment]
Agree I have the same issue. On an Unexpected Maker ProS3 I confirmed it has 8MB of RAM, the GUI says there is no SPI RAM (maybe because its used?) and other msgs are wrong too. But I understand its a new board.

On an official devkit s3-c1 (N8R8) it says "flash memory size 8MB" and SPI RAM available Yes. However when running a program I have about 62K of RAM only.  I am running the correct and latest firmware for octal SPI.
